dll/win32/shell32/lang/en-US.rc

```diff
@@ -236,8 +236,8 @@ BEGIN
     EDITTEXT 14015, 58, 128, 170, 10, ES_LEFT | ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP | ES_AUTOHSCROLL
     LTEXT "", -1, 8, 144, 221, 1, SS_ETCHEDHORZ | WS_CHILD | WS_VISIBLE
     LTEXT "Attributes:", 14020, 8, 152, 46, 10
-    AUTOCHECKBOX "&Read-only", 14021, 58, 151, 70, 10
-    AUTOCHECKBOX "&Hidden", 14022, 126, 151, 70, 10
+    AUTOCHECKBOX "&Read-only", 14021, 58, 151, 68, 10
+    AUTOCHECKBOX "&Hidden", 14022, 126, 151, 55, 10
     AUTOCHECKBOX "&Archive", 14023, 181, 151, 70, 10
```

Comment:
Looks like the last checkbox width is bigger than the dialog width, but I can fix this later.